docs/application/native/tutorials/details/native-removed-history.md

```diff
@@ -0,0 +1,39 @@
+# Removed functions and enumerations of Tizen Native API
+
+This page describes the functions and enumerations that were removed in Tizen 4.0 and 5.0.

```

Comment:
It looks like this topic is going to list up all removed functions and enumerations not limited in from Tizen 4.0.
So, how about changing this sentence like `This topic describes the functions and enumerations that were removed`?

Reply:
OK. I've fixed it. I will raise new Pull Request.